packages/models-library/src/models_library/resource_tracker.py

```diff
@@ -187,10 +187,33 @@ class SpecificInfo(HardwareInfo):
     to store aws ec2 instance type."""
 
 
+class UnitExtraInfo(BaseModel):
+    """Custom information that is propagated to the frontend. Defined fields are mandatory."""
+
+    CPU: NonNegativeInt
```

Comment:
TIP: for RAM and VRAM, why not using https://docs.pydantic.dev/1.10/usage/types/#bytesize?? Note that it is an `int` but with more parsing capabilities and extra tooling

SEE https://github.com/pydantic/pydantic/blob/1.10.X-fixes/pydantic/types.py#L1092-L1143